### DISCUSSION TITLE
Help using personnal custom icons

### DISCUSSION THREAD

**Discussion by goldriver2** [elax46/custom-brand-icons#446] [Q&A]
Ok, I created 2 SVG icons with the specified spec and they can be open and viewed in chrome.

I then added
"RVjack-extend":[0,0,24.0,24.0,"M7.0265236,22.230506 v -0.470257 h 2.2900202 2.2900202 l -0.0121,-6.336722 -0.0121,-6.3367213 -0.19082,-0.023513 -0.190821,-0.023513 V 6.0536438 3.0675077 L 9.2090411,3.0552514 7.2173507,3.0429951 V 2.3145797 1.5861961 h 5.5337993 5.533799 v 0.7283836 0.7283832 l -1.99169,0.012256 -1.991691,0.012256 V 6.053611 9.0397471 l -0.190821,0.023513 -0.19082,0.023513 -0.01209,6.3367219 -0.0121,6.336722 h 2.194597 2.19461 v 0.470257 0.470258 H 12.655733 7.0265236 Z"],
        "RVjack-retract":[0,0,24.0,24.0, "M6.9643947,9.8567638 V 9.4279123 L 9.0969389,9.4167467 11.229483,9.4055811 V 8.9119321 8.4182648 L 11.058421,8.4049338 10.887363,8.3916028 V 5.6468446 2.9020704 H 8.9943029 7.1012423 V 2.2366926 1.5713151 h 5.2686387 5.268638 v 0.6653775 0.6653778 h -1.89306 -1.893061 v 2.7447742 2.7447738 l -0.171058,0.013331 -0.171058,0.013331 V 8.9119477 9.405615 l 2.132544,0.011166 2.132544,0.011166 V 9.8567638 10.285615 H 12.369881 6.9643947 Z"], 

in the .js file

I then replaced the .js file on my HA system with the new one, restarted HA BUT the created icons are not showing when entered in an entity icon ex: phu:RVjack-extend shows a blank icon

Any one have an idea what I could have done wrong ?

![RVjack-extend](https://github.com/elax46/custom-brand-icons/assets/7215561/86872d8d-d09c-4ac0-a416-f0e5c41d64bb)
![RVjack-retract](https://github.com/elax46/custom-brand-icons/assets/7215561/f61b6f86-8996-4850-a77c-4b036d3adb45)
[custom-brand-icons.zip](https://github.com/elax46/custom-brand-icons/files/11925745/custom-brand-icons.zip)

**Comment by elax46** [comment]
Hello @goldriver2  sorry for the delay in replying I did some tests and the first icon is perfectly visible the second RVjack-retract has a view box which is not 0,0,24.0,24.0

I advise you to check the browser cache if you don't see the first icon and to check the code of the second one
![Screenshot 2023-08-02 105835](https://github.com/elax46/custom-brand-icons/assets/55278049/023aac0f-b084-471d-a96a-a2957e3a7b45)
![Screenshot 2023-08-02 105912](https://github.com/elax46/custom-brand-icons/assets/55278049/62803a7e-fd9c-4d23-858b-fa4cbeb73a30)
